### ISSUE TITLE
Tabs - Mounted hooks Bug

### ISSUE THREAD

**Issue by TheGinJiCode** [vuetifyjs/vuetify#964]
### Steps to reproduce

![image](https://user-images.githubusercontent.com/29909936/28102102-ae3a7980-66ff-11e7-8df7-fe4ccbffdc90.png)

![image](https://user-images.githubusercontent.com/29909936/28102128-dd986c78-66ff-11e7-8848-041a64d452d3.png)

### Versions

vuejs: 2.3.4
vuetify: latest 0.13
os: windows 10
### What is expected ?
Mounted hook should not run as many times as the number of tabs

### What is actually happening ?
Note: I only have 2weeks exp in vuejs and new to github :)

So basically its just a page with 7 tabs. Each Tabs is a router link and it will display below the tabs, just as what the template has stated. 

each route has a mounted hook that runs an axios request get to populate my datatables.
The funny part is that the mounted hook will be called as many times as the tabs. Ex. if I have 2 tabs, the mounted hook will trigger 2 times. :) 

Any Idea on whats the bug in this? I just thought that it has something to do with the tabs.

**Comment by TheGinJiCode** [comment]
So, I figure out that its because of the v-tabs-content. Each tab has it's own tabs-content. so apparently the bug is there. 

I did a quick fix in my templates and did this.

![image](https://user-images.githubusercontent.com/29909936/28103511-accef86a-6708-11e7-85f1-03379a66cd15.png)

I got what I want but, I don't know if it was the right approach there.

**Comment by dohomi** [comment]
@TheGinJiCode this is a lot of code to look at :) can you simply post this into a codepen?

**Comment by johnleider** [comment]
When creating an issue you should be provided with boilerplate for the ticket. This is important for myself and others to properly address your issue. Please reopen this with the required information, thank you.
